### ISSUE TITLE
Export error -- "panic: interface conversion: interface {} is nil, not string"

### ISSUE THREAD

**Issue by WhiteJP** [empiricaly/empirica#568]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

After running experiment locally, I tried to run `empirica export`. I get the error shown below

### Steps To Reproduce

1. create new test experiment `empirica create test-exp`
2. copy `tajriba.json` (below) to new `.empirica/local/tajriba.json`
3. run `empirica export` in `test-exp` directory. 
4. get error. 

### Empirica Version

```text
Version: v1.11.3
SHA:     c9e8647
Build:   218
Branch:  main
Time:    2024-07-15T01:58:21Z

Client:  1.11.3
Server:  1.11.3
```

### What OS are you seeing the problem on?

Windows

### What browser are you seeing the problem on?

Does not apply

### Relevant log output

```shell
15:27:23.269 INF Starting CSV export... output=/home/jpwhite/GitHub/others/test-exp/test-exp-2024-07-15-15-27-23.zip tajriba=/home/jpwhite/GitHub/others/test-exp/.empirica/local/tajriba.json
panic: interface conversion: interface {} is nil, not string

goroutine 1 [running]:
github.com/empiricaly/empirica/internal/export.prepare({0xc00012e820?, 0xc000170800?})
        /build/internal/export/export.go:94 +0x1a25
github.com/empiricaly/empirica/internal/export.ExportCSV({0xc00012e820?, 0xd85d5e?}, {0xc00012e910, 0x45})
        /build/internal/export/csv.go:16 +0x52
github.com/empiricaly/empirica/cmds/empirica/cmd.addExportCommand.func1(0xc0001a3180, {0x1515bc0, 0x0, 0xd72271?})
        /build/cmds/empirica/cmd/export.go:89 +0x447
github.com/spf13/cobra.(*Command).execute(0xc0001a3180, {0x1515bc0, 0x0, 0x0})
        /go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:872 +0x6aa
github.com/spf13/cobra.(*Command).ExecuteC(0xc00051d400)
        /go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:990 +0x38d
github.com/spf13/cobra.(*Command).Execute(...)
        /go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:918
github.com/empiricaly/empirica/cmds/empirica/cmd.Execute()
        /build/cmds/empirica/cmd/root.go:144 +0x236
main.main()
        /build/cmds/empirica/main.go:10 +0xf
```

### Anything else?

Here is the tajriba file that gets the error: [tajriba.json](https://github.com/user-attachments/files/16239494/tajriba.json). 
I have tried it different times with different tajriba files and it doesn't work for any. Interesingly, `empirica export` did work previously on my project. But as far as I can tell nothing has changed :/

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by npaton** [comment]
A fix for this will be in the next release.

**Comment by npaton** [comment]
Should be fixed in version 1.11.4.

**Comment by WhiteJP** [comment]
Thanks! I just updated to the latest version, checked again and everything works :)
